data/patches/miq_application.js.diff

```diff
@@ -7,7 +7,7 @@
 +// ---------------
 +
 +// Initialize MiqQE
-+// Make sure to update version here and in pytest_selenium when changed
++// Make sure to update version when changed
```

Comment:
Ironically, when only changing comments, we don't need to do anything, like in this case.